### DISCUSSION TITLE
Copilot, online course, workshop, python version, and collaboration

### DISCUSSION THREAD

**Discussion by haipinglu** [pykale/pykale#328] [Ideas] [answered]
Topics to discuss
- Copilot
- Online course via https://jupyterbook.org/en/stable/start/your-first-book.html & TA: https://pykale.github.io/transparentML, like https://github.com/wfdb/mimic_wfdb_tutorials
- Software Workshop
- Python version to use as main
- Ox CHI colab
- Overleaf

**Comment by xianyuanliu** [comment, marked as answer]
Discussed.
